### ISSUE TITLE
GSDP/Daashboard CSS breaks bootstrap-select 1.7.4

### ISSUE THREAD

**Issue by egroenen** [timcreative/get-shit-done-pro#26]
Hi,

I had to upgrade bootstrap-select to the latest version (1.7.4), it doesn't work with GSDP or the light dashboard which are adding the following CSS:

```
.dropdown-menu {
    visibility: hidden;
    margin: 0;
    padding: 0;
    border-radius: 10px;
    display: block;
    z-index: 9000;
    position: absolute;
    opacity: 0;
    filter: alpha(opacity=0);
    -webkit-box-shadow: 1px 2px 3px rgba(0, 0, 0, 0.125);
    box-shadow: 1px 2px 3px rgba(0, 0, 0, 0.125);
}
```

The `display: block` causes the drop-down to overlap other inputs below it with an invisible mask making them inaccessible. Bootstrap is using `display:none`, the solution was to not set the `display` attribute at all and let the bootstrap value filter up. I think you should try to avoid setting CSS attributes that cover the functionality if you can and stick to the presentation.

_dropdown.scss:

```
.dropdown-menu{
    visibility: hidden;
    margin: 0;
    padding: 0;
    border-radius: $border-radius-extreme;
    /* display: block; */
    z-index: 9000;
    position: absolute;
```

**Comment by alexandru-paduraru** [comment]
@egroenen thank you for pointing out, we checked and you are right, it looks like when we added the CSS for the Dropdown Animation we changed the default value. Can you please try to replace the ".dropdown-menu" with ".dropdown .dropdown-menu" and see if that is fixing the problem?

**Comment by alexandru-paduraru** [comment]
Fixed in v1.4.2 for GSDK Pro
